tests/integration/golang/aim/run/search_test.go

```diff
@@ -237,6 +241,35 @@ func (s *SearchTestSuite) Test_Ok() {
 		request request.SearchRunsRequest
 		runs    []*models.Run
 	}{
+		{
+			name: "SearchFirstPage",
+			request: request.SearchRunsRequest{
+				Query: `run.archived == True or run.archived == False`,
+				Limit: 2,
+			},
+			runs: []*models.Run{
+				run3,
+				run4,
```

Comment:
could you please change the order to the correct one here and in `SearchSecondPage`?
we can then fix the comparison logic in another PR

Reply:
ahhh

Reply:
you mean:
```
run3,
				run4,
```

Reply:
but should be:
```
run4,
run3
```

Reply:
?

Reply:
if so, then yep, it is just how it works if you check.

Reply:
yep, it should `run4, run3` in the order that they are returned

Reply:
no, this is how test works.

Reply:
the whole decoder has been messed up since I added it — there is supposed to be an order to the objects being streamed, which we lost. and also the output of `fml decode` has become unreadable

Reply:
I just tested locally and the tests are happy whether I specify `[run4, run3]` or `[run3, run4]` — this means we are _not_ verifying the order of the runs being returned and this is wrong
again, this doesn't have to be addressed in this PR at all — I would just like us to have the right expected order here so when we add order validation we don't have to change it anymore

Reply:
yes, there is a problem in this test. how we fetch data from the decoded response.

Reply:
changed ordering now.

Reply:
I've create #514 and #515 to address this